Tests/ComposableArchitectureTests/AppStorageTests.swift

```diff
@@ -253,13 +261,21 @@ final class AppStorageTests: XCTestCase {
   func testWillEnterForegroundFromBackgroundThread() async throws {
     @Shared(.appStorage("count")) var count = 0
 
-    let publisherExpectation = expectation(description: "publisher")
+    let publisherExpectation: XCTestExpectation = expectation(description: "publisher")
```

Comment:
can this be deleted?

```suggestion
    let publisherExpectation = expectation(description: "publisher")
```